### ISSUE TITLE
FeedbackChallenge : Smart Contract Audit Token SCAT DAO 

### ISSUE THREAD

**Issue by stephen-rowan** [Quality-Assurance-DAO/F5-Developer-ecosystem-Proposal#70]
## FeedbackChallenge : Smart Contract Audit Token SCAT DAO 

In the spirit of Project Catalyst's #FeedbackChallenge we are assessing to what extent the Catalyst Funding Process and it's Proposals are **open source** as part of the proposal [Quality-Assurance-DAO](https://cardano.ideascale.com/a/dtd/Quality-Assurance-DAO/351678-48088). Your proposal has been selected because "open source" is mentioned in it's text or comments.

We would appreciate it if you could respond to the following questions (in the reply to my comments below) : 

1.  **Open** - Is your Organisation open-source ?
1.  **Accessible** -  Can community contributors issue pull requests or raise issues on your source code ?
1.  **Transparent** - Can community contributors access and propose changes to the source text of your documentation ?

To keep the assessment transparent each comment is recorded as a [GitHub Issue]( https://github.com/Quality-Assurance-DAO/F5-Developer-ecosystem-Proposal/issues/70 ) on **Quality-Assurance-DAO's** proposal repository (Apache-2.0 License) where a more detailed explanation can be found. 

Together we can build a community of open source contributors on the Cardano platform.

Thank You 

Stephen Whitenstall, QA-DAO

## Detailed explanation 

Your [Issue reference](https://github.com/Quality-Assurance-DAO/F5-Developer-ecosystem-Proposal/issues/70) includes a more detailed explanation of the methodology of this FeedbackChallenge : Open Source. There is also a *Open Source Assessment* [Project Board](https://github.com/Quality-Assurance-DAO/F5-Developer-ecosystem-Proposal/projects/3) and a GitHub bot post Issues in my channel #quality-assurance-dao on Project Catalyst's Discord.

We believe this is important because many developers are concerned with the lack of a clear commitment to open source and fear their free contributions will be locked into a proprietary platform.

## Open Source Assessment Method
- [x] Draft a basic Open Source Assessment Method.
https://github.com/Quality-Assurance-DAO/DAO-Open-Source/tree/main/Documents/Assessments

### Basic Open Source Assessment
1.  **Open** - Is your Organisation open-source ?
1.  **Accessible** -  Can your users issue pull requests or raise issues on your organisation's source code ?
1.  **Transparent** -  Can your users access and propose changes to the source of your organisation's documentation ?

### Method
- [x] Do text search of "open source" on Catalyst Projects.
- [x] Post assessment questions in comments of proposal.
- [x] Raise Issue for each proposal's assessment.
- [x] Record response to assessment.
- [ ] Close Issue once proposal submitted and record responses

### Report
- [ ] Collate and publish report of findings with a Creative Commons License

**Comment by stephen-rowan** [comment]
Eric Helms (@erkoloco) kudos icon + 25 minutes ago
Idea Submitter

> swhitenstall Hi Stephen, thanks so much for taking the time to leave a comment. Currently, this is an idea I have been developing and there has been no code written. I am hoping that if it is popular in Catalyst, I will be able to move to that next stage. But to answer your questions, Yes I expect everything that we do to be open source, accessible, and transparent. Everything that we create for this project, including the code written to govern the DAO, the audit program, training materials, etc will be available for the community. Part of our success will be measured by improving the audit environment for Cardano overall, giving minimum standards to be achieved, and getting everyone thinking about Dapp security in the same way. I hope that answers your question. If you would like to know anything else, please ask.

**Comment by stephen-rowan** [comment]
Thank you for your response, Eric Helms (erkoloco) . It has helped a lot.

I have captured them at [ https://github.com/Quality-Assurance-DAO/F5-Developer-ecosystem-Proposal/issues/70 ] and moved the status of the Issue to "Done".

Note : The Issue can still be altered and commented upon via GitHub until Fund 5 Proposals are finalized.

As I set out in the "Detailed explanation" section of each Issue : I will collect all data in a report of findings to be published on GitHub with a Creative Commons License after Fund 5 Proposals are finalized.

Good Luck with your proposal.

Stephen Whitenstall QA-DAO
